web/app/templates/base.html

```diff
@@ -30,6 +31,16 @@
   {{ moment.include_moment() }}
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script>
   <script type="text/javascript">
+  
     var $SCRIPT_ROOT = {{ url_for('main.index', _external = True)|tojson|safe }}
+    var csrftoken = $('meta[name=csrf-token]').attr('content')
```

Comment:
I used the method described in the wtforms documentation, but we surely can discuss it. 
https://flask-wtf.readthedocs.io/en/latest/csrf.html